src/core/scene/a-scene.js

```diff
@@ -320,7 +364,16 @@ module.exports = registerElement('a-scene', {
         if (this.isPlaying) {
           this.tick(time, timeDelta);
         }
-        this.effect.render(this.object3D, camera);
+        // Postproc must be explicitly enabled by a component or system.
```

Comment:
At minimum there needs to be a way to instruct a-scene to render to texture instead of directly to the canvas. Tocks are meant to execute after the base render, so they're not really useful for updating the scene at that point. We could go with @fernandojsg suggestion and just check for the presence of tock components but then the developer would have to constantly add and remove a bunch of components. The .enablePostProcessing flag is purely for convenience